### ISSUE TITLE
A11y critical regression: navigation on iOS with VoiceOver in Article list is broken

### ISSUE THREAD

**Issue by markcartertm** [Ranchero-Software/NetNewsWire#5096]
There is a critical accessibility regression in the latest iOS build of NetNewsWire. Up until the latest build with screen reader visually impaired user using voiceover. I could flip one finger right to read the next article in the article list. Voiceover would read the name of the feed, followed by the article and would let me know if it's red or unread. Now I suddenly need to navigate multiple times through one article and new elements are red that are completely not useful. Some change in the Article list view broke the ability to navigate with a screen reader which renders NetNewsWire extremely painful to visually impaired users on IoS. Experienced using today view on latest iOS with Net NetNewsWire 7.0 (Build 7024)

**Comment by adincebic** [comment]
Just noticed the same.

**Comment by stuartbreckenridge** [comment]
This should be resolved with the #5104 PR
